agent/util-scripts/pbench-tool-meister-start

```diff
@@ -272,44 +321,111 @@ def main(argv):
     sh.setFormatter(shf)
     logger.addHandler(sh)
 
-    try:
-        group = argv[1]
-    except IndexError:
-        group = "default"
+    # +
+    # Step 1 - Load the tool group data for the requested tool group
+    # -
 
-    # 1. Load the tool group data given the tool group argument
+    # Verify all the command line arguments
+    group = parsed.tool_group
     try:
+        # Load the tool group data
         tool_group = ToolGroup(group)
+    except BadToolGroup as exc:
+        logger.error(str(exc))
+        return 1
     except Exception:
-        logger.exception("failed to load tool group data")
+        logger.exception("failed to load tool group data for '%s'", group)
         return 1
 
-    # See if anybody told us to use certain options with SSH commands.
-    ssh_opts = os.environ.get("ssh_opts", "")
+    sysinfo = parsed.sysinfo
+    if sysinfo is None:
+        sysinfo = ""
+    else:
+        sysinfo_l, bad_l = verify_sysinfo(sysinfo)
+        if bad_l:
+            logger.error('invalid sysinfo option(s), "{}"', ",".join(bad_l))
+            if not sysinfo_l:
+                return 2
```

Comment:
Removed the check and `return 2` entirely.

Prior to this commit, each "bench script" would call `pbench-collect-sysinfo --sysinfo=${sysinfo} --check` via `verify_common_bench_script_options` in `agent/base`.  So the `bench-script` would exit before the normal call to `pbench-collect-sysinfo` would be made.  That "normal" call did not check the return value from `pbench-collect-sysinfo`.

The bench scripts still check the `${sysinfo}` option via `verify_common_bench_script_options` in `agent/base`.

So by the time `pbench-tool-meister-start` is now called by a bench script, the `${sysinfo}` option value has already be checked.  This log message really just needs to emit any errors that are encountered (in case somebody is calling `pbench-tool-meister-start` without first checking the `${sysinfo}` argument).